pkgs/by-name/yt/ytui-music/package.nix

```
@@ -0,0 +1,59 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, openssl
+, sqlite
+, stdenv
+, darwin
+, mpv
+, youtube-dl
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ytui-music-unstable";
+  # Upstream doesn't currently have tags that would be any more meaningful than date of commit
+  # Change this when they upstream has a proper release
+  version = "18-09-2023";
```

upstream does have tags

```suggestion
  version = "v2.0.0-rc1";
```

---

I could use them with this however their usefulness in the way that I had wanted to use the field in the rest of the derivation (installChecks, passthru etc) were questionable. However as those are more optional where as the use of version in the `src` is more traditional and standard I can add the tags upstream provides.

---

fixed in https://github.com/NixOS/nixpkgs/pull/256702/commits/66733f9113ecfd9f63910008c2056bd3767d3676